index.html

```diff
@@ -37,5 +37,161 @@ <h3>LET'S CONNECT</h3>
     </div>
 
 
+    <section id="portfolio" class="card">
```

Comment:
- [ ] Please 🙏🏽 use this class in the CSS file to implement the display grid for the projects section to match all the project requirements.

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). **Use CSS Grid in the main container that contains all of the cards** and use Flexbox to lay out an individual card.